consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumerTest.java

```diff
@@ -325,17 +299,17 @@ private IIndividualRequest createIndividualRequest() {
         bulkSubmission.setId(1L);
         bulkSubmission.setNumberOfRequest(1);
 
-        final IIndividualRequest individualRequest = new IndividualRequest();
-        individualRequest.setSdtRequestReference("Test");
+        final IIndividualRequest iindividualRequest = new IndividualRequest();

```

Comment:
Can leave this variable name as individualRequest - other interface variables in this method don't bother with the i prefix.

Reply:
The ii prefix makes the variable unique to the method and differs from the global method - which albeit will be populated when returned from this method.